runelite-api/src/main/java/net/runelite/api/WorldType.java

```diff
@@ -61,17 +61,17 @@
 	 */
 	TOURNAMENT(1 << 25),
 	/**
-	 * Deadman Tournament world type.
+	 * Deadman Tournament world type (unused).
```

Comment:
This doesn't seem like this should be included in the first commit, or at all really since the second commit should remove it.